roles/contiv_network/tasks/main.yml

```diff
@@ -107,6 +107,10 @@
   retries: 9
   delay: 10
 
+# get the netmaster status for debugging in case there are errors
+- name: get netmaster status
+  shell: systemctl status netmaster -l
+
```

Comment:
How does it different that 

https://github.com/contiv/ansible/blob/master/roles/contiv_network/tasks/main.yml#L103